src/octoprint/plugins/eventmanager/templates/eventmanager_settings.jinja2

```diff
@@ -12,7 +12,7 @@
 <div data-bind="foreach: settingsViewModel.settings.plugins.eventmanager.subscriptions" class="row-fluid">
     <div class="row-fluid">
         <div class="span4" data-bind="text: event"></div>
-        <div class="span6" data-bind="text: command"></div>
+        <div class="span6" data-bind="attr: {title: command}, text: custom_name"></div>
```

Comment:
When people update, they may be confused by the fact that it is now blank where there once was the command they had configured. I would suggest using the command here if no name has been configured.

I think this should do it 👇 

```suggestion
        <div class="span6" data-bind="attr: {title: command}, text: custom_name() ? custom_name : command"></div>
```

Reply:
It wont be blank because there is a migration process that will populate the custom_name once the plugin update. Rather than do that on the frontend, I opted to take care on the backend to ensure structure to be the same.